### PR TITLE
Music: play together variant

### PR DESCRIPTION
**Pull request by breville**
This adds a variant of the `play together` block, designed to be used in start sources, which doesn't contain a "[next connection](https://developers.google.com/blockly/guides/create-custom-blocks/define-blocks#next_connection)".

Screenshot 2024-10-06 at 3 37 34 PM

### CODE REVIEW

**Review by thomasoniii — APPROVED** [review]
lgtm 🚀

File: apps/src/music/blockly/toolbox/toolboxBlocks.ts [hunk 1]
```diff
@@ -168,6 +168,12 @@ const toolboxBlocks: {[blockType in BlockTypes | string]: BlockInfoWithText} = {
     kind: 'block',
     type: BlockTypes.PLAY_SOUNDS_TOGETHER,
   },
+  [BlockTypes.PLAY_SOUNDS_TOGETHER_NO_NEXT]: {
+    levelbuilderText: 'Play Together',
```
Comment (sanchitmalhotra126) [inline apps/src/music/blockly/toolbox/toolboxBlocks.ts]: Update this to highlight the difference?

**Review by sanchitmalhotra126 — APPROVED** [review]

File: apps/src/music/blockly/toolbox/toolboxBlocks.ts (hunk 1 above)
Comment (breville) [reply apps/src/music/blockly/toolbox/toolboxBlocks.ts]: Good call.  Updated.

**Comment by dju90** [comment]
This is cool!

**Pull request merged**